Utilities/Mergers/src/MergerBuilder.cxx

```diff
@@ -96,8 +97,13 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
     merger.algorithm = framework::adaptFromTask<FullHistoryMerger>(mConfig, subSpec);
   }
 
-  merger.inputs.push_back({"timer-publish", "TMR", mergerDataDescription(mName), mergerSubSpec(mLayer, mId), framework::Lifetime::Timer});
-  merger.options.push_back({"period-timer-publish", framework::VariantType::Int, static_cast<int>(mConfig.publicationDecision.param * 1000000), {"timer period"}});
+  // Create the TimerSpec for cycleDurations
+  std::vector<o2::framework::TimerSpec> timers;
+  for (auto& [cycleDuration, period] : mConfig.publicationDecisionNew.param) {
```

Comment:
`period` is rather confusing, can you use sth like `durationSinceStart`?

Reply:
or `validity`, as in the DPL interface

Reply:
ok for validity